src/cpp/include/openvino/genai/visual_language/pipeline.hpp

```diff
@@ -18,15 +21,26 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
     /// @param model_dir A folder to read tokenizer and model IRs.
     /// @param device Inference device. A tokenizer is always compiled
     /// for CPU.
-    /// @param device_config A config to pass to ov::Core.set_property()
-    /// and ov::Core::compile_model().
-    /// @param core ov::Core instance to use.
-    explicit VLMPipeline(
-        const std::filesystem::path& model_dir,
```

Comment:
> Maybe we can perform alignment to opposite side?

Yeah, I explain having `std::string` with our inertia after C++11. `std::filesystem::path` is preferred. It may even be that it would let `pybind` to take `Path` as well.

Reply:
done